src/seth/libexec/seth/seth-call

```diff
@@ -70,6 +70,7 @@ else
   opts+=(--block     "${ETH_BLOCK:-$(seth --field number <<< "$block")}")
   opts+=(--number    "${ETH_NUMBER:-$(seth --field number <<< "$block")}")
   opts+=(--timestamp "${HEVM_TIMESTAMP:-$(seth --field timestamp <<< "$block")}")
+  opts+=(--chainid   "${ETH_CHAINID:-$(seth rpc eth_chainId | seth --to-dec)}")
```

Comment:
@MrChico are you satisfied?

Reply:
no, can you add it to seth-run-tx too plx